woocommerce/woocommerce-android
# Feature/9930 new blaze view my store

Closes: #9930


### Description

Add new UI Blaze view to My Store tab. This new Blaze view is displayed when the user has no other active Blaze campaigns (note the display logic is out of the scope of this PR and not implemented)

### Testing instructions


Just run the app and check the displayed Blaze campaign view matches the Figma design: mBhyoV7OPhFnOugxLy6SMS-fi-633%3A35155

### Images/gif


![Screenshot 2023-10-12 at 09 12 48](https://github.com/woocommerce/woocommerce-android/assets/2663464/e0493406-2111-4f63-8766-ba97df6d2d7b)
